b2sdk/urllib3/NOTICE

```diff
@@ -0,0 +1,8 @@
+Urllib3
```

Comment:
this module name says that it vendors urllib3 when in reality it vendors botocore and not urllib3 at all

botocore patches both urllib3 and requests

suggested name `_botocore`.
* botocore since it is botocore code
* `_` since we don't want anybody EVER think they should do `import b2sdk._botocore` (apiver already forbids this, but `_` prefix is more well known convention than apiver